### PR TITLE
HEXDEV-628: don't return JSON exceptions when REST API clients ask ab…

### PR DESCRIPTION
**Pull request by rpeck**
…out Frames with deleted Vecs.  This will allow the clients to keep working with functional Frames.

---
This change is [Reviewable](https://reviewable.io/reviews/h2oai/h2o-3/289)

### CODE REVIEW

**Comment by bgallmeister** [comment]
Needed for ADP I believe.  Please accelerate review process...

File: h2o-core/src/main/java/water/api/FramesHandler.java [hunk 1]
```diff
@@ -266,8 +266,14 @@ public FramesV3 summary(int version, FramesV3 s) {
 
     if( null != frame) {
       Futures fs = new Futures();
-      for( Vec v : frame.vecs() )
-        v.startRollupStats(fs, Vec.DO_HISTOGRAMS);
+      int i = 0;
```
Comment (mmalohlava) [inline h2o-core/src/main/java/water/api/FramesHandler.java]: Why the code is duplicating code in FrameV3?

**Comment by mmalohlava** [comment]
Why are the check failing?

**Comment by abal5** [comment]
failure was due to missing cus-data and test has been changed to nopass for now on runit_small please... resubmit

**Pull request merged**